src/components/NotePage/NoteList/NoteItem.tsx

```diff
@@ -118,53 +118,92 @@ export default ({
     (event: React.MouseEvent) => {
       event.stopPropagation()
       event.preventDefault()
-      popup(event, [
-        {
-          type: MenuTypes.Normal,
-          label: t('note.duplicate'),
-          onClick: async () => {
-            createNote(note.storageId, {
-              title: note.title,
-              content: note.content,
-              folderPathname: note.folderPathname,
-              tags: note.tags,
-              bookmarked: false,
-              data: note.data
-            })
-          }
-        },
-        {
-          type: MenuTypes.Normal,
-          label: t('note.delete'),
-          onClick: async () => {
-            if (!note.trashed) {
-              trashNote(note.storageId, note._id)
-            } else {
-              messageBox({
-                title: t('note.delete2'),
-                message: t('note.deleteMessage'),
-                iconType: DialogIconTypes.Warning,
-                buttons: [t('note.delete2'), t('general.cancel')],
-                defaultButtonIndex: 0,
-                cancelButtonIndex: 1,
-                onClose: (value: number | null) => {
-                  if (value === 0) {
-                    purgeNote(note.storageId, note._id)
+
+      let menuItems: MenuItem[]
```

Comment:
Rather than using `let` and `if` statement, I would use `const` and a ternary operator.

```ts
const menuItems: MenuItem[] = note.trashed ? [...] : [...]
```

Reply:
Thanks to fix my code:pray: